tests/components/mold_indicator/test_sensor.py

```diff
@@ -50,7 +55,7 @@ def test_setup(self):
 
         moldind = self.hass.states.get("sensor.mold_indicator")
         assert moldind
-        assert "%" == moldind.attributes.get("unit_of_measurement")
+        assert UNIT_PERCENTAGE == moldind.attributes.get("unit_of_measurement")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093397)